bridges/chains/chain-cumulus/Cargo.toml

```diff
@@ -26,6 +26,7 @@ frame-support = { workspace = true }
 frame-system = { workspace = true }
 sp-api = { workspace = true }
 sp-std = { workspace = true }
+parachains-common = { workspace = true }
```

Comment:
This should also be added bellow in `std = [...]`

Reply:
Oops, done.